incremental-font-transfer/src/glyph_keyed.rs

```diff
@@ -449,18 +461,26 @@ fn patch_offset_array<'a, T: GlyphDataOffsetArray>(
     }
 
     // Step 3: add new tables to the output builder
-    offset_array.add_to_font(font_builder, new_data, new_offsets)?;
+    offset_array.add_to_font(font_builder, new_data, new_offsets, new_offset_type)?;
 
     Ok(())
 }
 
 /// Classifies the different style of offsets that can be used in a data offset array.
+#[derive(Clone, Copy, PartialEq, Eq)]
 enum OffsetType {
     Long,
     ShortDivByTwo,
 }
 
 impl OffsetType {
+    fn max_representable_size(&self) -> u64 {
```

Comment:
because this is a copyable type, it's various methods could take `self` by value instead of by reference, but also nbd.